content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -1094,7 +1094,7 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
 3. Remove `v1beta1` from the CustomResourceDefinition `status.storedVersions` field.
 
 {{< note >}}
```

Comment:
I think we could simplify the contents of this note to:
<pre>

Here is an example of how to patch the `status` subresource for a CRD object using `kubectl`:

```bash
kubectl patch customresourcedefinitions &lt;CRD_Name&gt; --subresource='status' --type='merge' -p '{"status":{"storedVersions":["v1"]}}'
```
</pre>
WDYT?

Reply:
Yeah, I think it's reasonable. Previously `--subresource` was new, so we needed explicit info when it's available. But since it's beta for all supported releases, we can just mention the example only.